src/docs/asciidoc/http-client.adoc

```diff
@@ -429,6 +429,42 @@ public class Application {
 ----
 ====
 
+
+== Bootstrap Configuration 
+
+Use `HttpClient.from` and create a `TcpClient` when you need more control of your client configuration, i.e., when you 
+want to specify the local address.
+
+
+====
+[source,java]
+----
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.tcp.TcpClient;
+import java.net.InetSocketAddress;
+
+public class Application {
+
+    public static void main(String[] args) {
```

Comment:
nitpick: mix of spaces and tabs. you can use tabs everywhere

Reply:
actually we use spaces in the documentation, but I agree with @simonbasle, please correct it so that it uses only spaces and not a mixture of both